test/FSharpTestProjects/TestApp/project.json

```diff
@@ -10,11 +10,11 @@
     "dependencies": {
         "TestLibrary": { "target": "project" },
         "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-151221",
-        "System.IO": "4.0.11-beta-23428",
-        "System.Console": "4.0.0-beta-23428",
-        "System.Runtime": "4.0.21-beta-23428",
-        "System.Diagnostics.Process": "4.1.0-beta-23428",
-        "Microsoft.NETCore.Runtime": "1.0.1-beta-23428"
+        "System.IO": "4.0.11-rc2-*",
```

Comment:
because fsharp.core require at least rc2.
btw i see 3d493f7d9d840baa1c2cb7575b4811f3db791bc7 is committed in `master`, i'll do like `test/TestProjects/*/project.json`, with 

```json
{
   "dependencies": {
        "TestLibrary": "1.0.0-*",

        "NETStandard.Library": "1.0.0-rc2-23704",
        "Microsoft.NETCore.Platforms": "1.0.1-rc2-23704"
 }
}
```